feature/platform/tests/telemetry_inventory_test/telemetry_inventory_test.go

```diff
@@ -133,6 +135,7 @@ func TestHardwareCards(t *testing.T) {
 				serialNoValidation:    true,
 				mfgNameValidation:     true,
 				mfgDateValidation:     false,
+				modelNameValidation:   true,
```

Comment:
is chassis the only component type you care about at this time?  Not linecard or others?

Reply:
Got it, thanks!